lib/Watching.js

```diff
@@ -149,6 +149,9 @@ class Watching {
 			}
 			for (const cb of this.callbacks) cb();
 			this.callbacks.length = 0;
+			if (this.closed) {
```

Comment:
This can be removed. afterDone should be called after each run.

Reply:
Got it.